src/app/home-page/widget/starred-box/starred-box.component.html

```diff
@@ -1,192 +1,204 @@
 <mat-card class="h-100">
-  <div id="header-bar" fxLayout="row" fxLayoutAlign="space-between">
```

Comment:
a lot of whitespace changes in here, is prettier acting up https://github.com/dockstore/dockstore-ui2/blob/develop/.prettierrc?

Reply:
I don't think so, I had to indent everything for the new div I added